src/es/components/molecules/teaser/Teaser.js

```diff
@@ -0,0 +1,143 @@
+ *
+ * @export
+ * @attribute {namespace} namespace
+ * @attribute {color} color
+ * @type {CustomElementConstructor}
+ */
+export default class KsTeaser extends Teaser {
+  constructor (options = {}, ...args) {
+    super({
+      importMetaUrl: import.meta.url,
+      ...options
+    }, ...args)
+  }
+
+  /**
+   * fetches the template
+   *
+   * @return {Promise<void>}
+   */
+  fetchTemplate () {
+    switch (this.getAttribute('namespace')) {
+      case 'teaser-tile-':
+        return this.fetchNamespaceTemplate(['tile-/tile-.css'])
+      case 'teaser-story-':
+        return this.fetchNamespaceTemplate(['story-/story-.css'])
+      default:
+        return super.fetchTemplate()
+    }
+  }
+
+  /**
+   * fetches the actual teaser namespace template
+   *
+   * @return {Promise<void>}
+   */
+  fetchNamespaceTemplate (relativePaths) {
+    const baseStyles = [
+      {
+        path: `${this.importMetaUrl}../../web-components-toolbox/src/css/style.css`,
+        namespace: false
+      }
+    ]
+
+    const allStyles = relativePaths.map((relativePath) => ({
+      // @ts-ignore
+      path: `${this.importMetaUrl}${relativePath}`,
+      namespace: false
+    })).concat(baseStyles)
+
+    return this.fetchCSS(allStyles).then(fetchCSSParams => {
+      // make template ${code} accessible aka. set the variables in the literal string
+      fetchCSSParams[0].styleNode.textContent = eval('`' + fetchCSSParams[0].style + '`')// eslint-disable-line no-eval
+    })
+  }
+
+  /**
+   * renders the m-Teaser css
+   *
+   * @return {Promise<void>}
+   */
+  renderCSS () {
+    this.css = /* css */`
+        :host * {
+          box-sizing: border-box;
+          margin: 0 !important;
+        }
+
+        :host a-picture {
+          display: block;
+          overflow: hidden;
+        }
+
+        :host figure {
+          display: flex;
+          flex-direction: column;
+          color: var(--color);
+        }
+
+        :host figcaption {
+          display: flex;
+          flex-direction: column;
+          gap: 0.75rem;
+          flex-shrink: 0;
+          width: 100%;
+        }
+
+        :host figcaption > * {
+          width: 100%;
+        }
+
+        :host figcaption > strong:first-child {
+          display: block;
+          color: var(--color-${this.getAttribute('color')}, black);
+          font-family: var(--pretitle-font-family);
+          font-size: var(--pretitle-font-size);
+          font-weight: var(--pretitle-font-weight);
+          line-height: var(--pretitle-line-height);
+          transition: var(--transition);
+        }
+
+        :host figcaption h5,
+        :host figcaption h4,
+        :host figcaption h3,
+        :host figcaption h2,
+        :host figcaption h1 {
+          color: inherit !important;
+          transition: var(--transition);
+          font-family: var(--headline-font-family) !important;
```

Comment:
ist hier !important wirklich notwendig? Ist zwar okay, da wir in der ShadowDom sind, waere aber schoener ohne.